### PR TITLE
Project setup and list structure

### PR DESCRIPTION
**Pull request by yghoor**

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: Linters
```
Comment (Cerdtruir) [inline .github/workflows/linters.yml:1]: did you remove the linters on the latest branch?

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+function displayItems(tasksArr) {
+  tasksArr.forEach((task) => {
+    const item = document.createElement('li');
+    item.id = `item-${task.index}`;
+    item.classList.add(`${task.completed}`);
+
+    item.innerHTML = `
+      <input type="checkbox">
+      <span>${task.description}</span>
```
Comment (Cerdtruir) [inline src/index.js:36]: its not a good idea to use inner html on user input, when you are using user input this should be changed

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+h2 {
+  padding: 0 14px;
+  margin: 12px 0;
+  display: flex;
+  justify-content: space-between;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.reload {
```
Comment (Cerdtruir) [inline src/style.css:25]: this class name can be more descriptive

**Review by Cerdtruir — APPROVED** [review]
It looks good, please make the changes then merge.

**Comment by Cerdtruir** [comment]
The memo readme also says it should be in production mode, but it looks like its in dev mode. Please check this as well

**Comment by yghoor** [comment]
Fixed issues in latest branch before test branches

**Pull request merged**